### PR TITLE
Styling Milestone

### PR DESCRIPTION
**Pull request by wuyepabdul**
In this milestone, I did the following:
- Added styles to the entire app
- Used the information detailed in Zeplin to style my app.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -29,21 +32,93 @@
...
+}
+
+.book-category {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
```
Comment (dasileker) [inline src/index.css:90]: - the design is good, but let's make look great.

- [x] the remove btn should be colored not having a bg-color.
- [x] also narrow the width if you will not add chapter and progress to it.
- [x] lastly, add edits and comment as the design has.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @wuyepabdul,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 2]
```diff
@@ -29,28 +32,111 @@
...
+}
+
+.book-btn:hover {
+  cursor: pointer;
+}
+
 .book,
 .book-buttons {
   padding-top: 20px;
 }
```
Comment (dasileker) [inline src/index.css]: - [x] Move this up before line 106 .book-buttons:last-child().

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @wuyepabdul,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- Please make sure to fix the linters and submit them for review, your project looks good.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by afanimartin — APPROVED** [review]
Hi @wuyepabdul, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by wuyepabdul** [comment]
Thanks for the review @dasileker and @afanimartin .

**Pull request merged**